lib/ansible/modules/network/ios/ios_ntp.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = '''
+---
+module: ios_ntp
+extends_documentation_fragment: nxos
+version_added: "2.8"
+short_description: Manages core NTP configuration.
+description:
+    - Manages core NTP configuration.
+author:
+    - Federico Olivieri (@Federico87)
+options:
+    server:
+        description:
+            - Network address of NTP server.
+    peer:
+        description:
+            - Network address of NTP peer.
+    source_int:
+        description:
+            - Source interface for NTP requests.  
+     state:
```

Comment:
```suggestion
    state:
```

Reply:
That should be just a doc string, right? :)